File: PetBooK.BL/DTO/DoctorUser.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PetBooK.BL.DTO
{
    public class DoctorUser
    {
        public string Name { get; set; }
        public string Degree { get; set; }

        public DateOnly HiringDate { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string Phone { get; set; }
        public string UserName { get; set; }
        public string Location { get; set; }
        public int? Age { get; set; }
        public string Sex { get; set; }
        public IFormFile Photo { get; set; } // to upload the image

    }
}

File: Front-end/PetBook/src/app/Models/doctor-user.ts
export class DoctorUser {
    constructor(
        public name: string,
        public email: string,
        public password: string,
        public phone: string,
        public userName: string,
        public location: string,
        public age: number|null,
        public sex: string,
        public degree: string,
        public hiringDate: Date,
        public photo: File|null

          )
          {}
      }
      
